source/content/guides/edge-integrations/01-introduction.md

```diff
@@ -32,16 +32,16 @@ The benefits of Edge Integrations:
 	- You can improve productivity by using existing content, style guides, media, and CMS integrations. No need to manage assets in multiple places.
 
 - Performance-forward
-	- Improve credibility with a fast, seamless customer experience- distribute personalized content across dozens of global and US points of presence.
+	- Improve credibility with a fast, seamless customer experience, and distribute personalized content across dozens of global and US points of presence.
 
 - Cost-effective
 	- Increase business impact by instrumenting and measuring success with your current analytics products. No need for new segmentation tooling.
 
 - Geographic targeting
-	- Based on the location of the visitor, the site will deliver a different homepage here for Canadian visitors vs US (Poutine vs Pizza).
+	- Based on the location of the visitor, the site can deliver a different homepage — e.g. Canadian visitors might see poutine, where US visitors would see pizza.
```

Comment:
I think it might be easier to read if we just used "for example" rather than `e.g.`. Also, I think "whereas" makes more sense grammatically than "where" in this case.

```suggestion
	- Based on the location of the visitor, the site can deliver a different homepage — for example, Canadian visitors might see poutine, whereas US visitors would see pizza.
```